vault/changelog.early-seed.md

```diff
@@ -2,40 +2,41 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1654869904786
+updated: 1656077960647
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.100.0
+## 0.101.0
+June 24, 2022
 
-Dendron 0.100 has sprouted 🌱
+Dendron 0.101 has sprouted 🌱
 
-**v100:** We will be releasing version 100 of Dendron next week! This will mark 100 releases and over 600 features shipped! We'll be launching on Product Hunt on Tuesday June 21st, 12:01AM PST. If Dendron has been of use to you, please leave feedback on our [product hunt page](https://www.producthunt.com/posts/dendron-v100) at that time - this would help us out a lot! 🙏
+**Depth Filter for Local Graph**: Interested in navigating beyond an active note's immediate parent, child or links in the local graph panel? With new depth filters, you can customize the graph view experience to also see an active note's grandchildren and grandparents. 
```

Comment:
**Depth Filter for Local Graphs**:  Want to see the bigger picture? The graph depth filter now lets you control how much context you see in the [[Local Graph View|dendron://dendron.dendron-site/dendron.topic.sidebar.local-graph-view]]

Reply:
updated